src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+
+public class GuiHandler
+{
+
+
+    private Pane pane;
+    /* Elementen voor het antwoord */
+    private Label lblSetAnswer;
+    private TextField txtSetAnswer;
+    private Button btnSetAnswer;
+    private Rectangle shapeLine;
+    /* Elementen voor het spel */
+    private Label lblRules, lblTryALetter, lblTryAWord, lblLettersCorrect, lblLettersWrong, gameEndText, guesWordLenght;
+    private TextField txtTryALetter, txtTryAWord;
+    private Button btnTryALetter, btnTryAWord, reset;
+    private ImageView imgHangman;
+
+    /* Variabelen */
+    private String answer;
+    private int mistakes;
+    private boolean gameRunningStatus;
```

Comment:
Kan je hier geen Enum van maken zodat je meer kan dan alleen false, true 
dingen zoals running, paused, error, crashed en wat je ook maar wil kan je dan toevoegen. 
en dit houd dus je status bij. 

```suggestion
    private GameState gameStatus;
    
    private enum GameState{
         Running, 
         NotRunning,
         Paused
   }
```